platform/ios/INSTALL.md

```diff
@@ -52,8 +52,6 @@ bash "${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/Mapbox.framework/strip-fra
 
 #### Static framework
 
-If your application targets iOS 7.x, you’ll need to install the static framework instead:
```

Comment:
I'd leave in a sentence just to make it clear that these are alternative instructions, not additive. "If you need a static framework…"